src/public/index.ts

```diff
@@ -0,0 +1,5 @@
+function main() {
```

Comment:
when does this code get executed?

Instead I would just register a backend API that return hello world